### ISSUE TITLE
TypeError: unsupported operand type(s) for -: 'float' and 'NoneType'

### ISSUE THREAD

**Issue by traumschule** [mikeperry-tor/vanguards#29]
Vanguards successfully connected to Tor, after ~15 minutes it shows:
```
INFO:stem:CIRC event had an unrecognized remote_reason (CHANNEL_CLOSED). Maybe a new addition to the control protocol? Full Event: 'CIRC 1521 FAILED $fingerprint,$fingerprint BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=TESTING TIME_CREATED=2018-08-25T02:12:40.252792 REASON=DESTROYED REMOTE_REASON=CHANNEL_CLOSED'
NOTICE[Sat Aug 25 04:13:57 2018]: The connection to guard $guard1 was closed with a live circuit.                                              
NOTICE:TorCtl:The connection to guard $guard1 was closed with a live circuit.                                                                  
Exception in thread Event Notifier:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib/python2.7/dist-packages/stem/control.py", line 973, in _event_loop
    self._handle_event(event_message)
  File "/usr/lib/python2.7/dist-packages/stem/control.py", line 3796, in _handle_event
    listener(event_message)
  File "/media/user/src/tor/vanguards/src/vanguards/bandguards.py", line 240, in circ_event                                                                              
    self.circuit_destroyed(event)
  File "/media/user/src/tor/vanguards/src/vanguards/bandguards.py", line 204, in circuit_destroyed                                                                       
    if event.arrived_at - self.guards[guardfp].killed_conn_at \
TypeError: unsupported operand type(s) for -: 'float' and 'NoneType'
```

**Comment by mikeperry-tor** [comment]
Yikes this is a bad crash bug. I reproduced it in a unittest and pushed a fix to master.

The fix will be in 0.2.2. Leaving this open until that release for reference.

**Comment by mikeperry-tor** [comment]
Ok v0.2.2 is released. Closing this.

**Comment by traumschule** [comment]
this was a quick fix! vanguards 0.2.2 is stable since some hours here, thanks!
> NOTICE:TorCtl:Vanguards 0.2.2 connected to Tor 0.3.5.0-alpha-dev using stem 1.6.0
